### ISSUE TITLE
[chromium] build fail

### ISSUE THREAD

**Issue by unicomp21** [JanitorTechnology/dockerfiles#83]
Built the docker image, but then couldn't build chromium.  Looks like there's a version mismatch.

**Comment by jankeromnes** [comment]
Hi @unicomp21, thanks a lot for reporting this issue! Could you please indicate the steps you took, and the exact error you got?

Chromium build is currently disabled because it takes very long, but it is supposed to work, so if you found a problem we should fix it.

**Comment by jankeromnes** [comment]
I've got this:

```
ninja: Entering directory `out/Default'
ninja: fatal: ninja version (1.5.1) incompatible with build file ninja_required_version version (1.7.2).
```

Maybe that's the same error you saw? Anyway, I'll fix it.

**Comment by jankeromnes** [comment]
So Ninja as installed with `sudo apt-get install ninja-build` on our Ubuntu 16.04 dockerfile is too old (like most other developer packages on Ubuntu... *sigh*).

We should install a more recent Ninja, maybe following https://github.com/ninja-build/ninja/blob/master/HACKING.md (I need to find out how/where to install the binary and docs).

**Comment by jankeromnes** [comment]
Should hopefully be fixed by a675a0e8eddfba6d4d3e07ea5901be2fb025248b.

**Comment by jankeromnes** [comment]
(The deployment to Docker Hub of the potentially fixed images is currently blocked by https://github.com/rust-lang-nursery/rust-clippy/issues/2073. Please stay tuned.)

**Comment by jankeromnes** [comment]
Fixed by a675a0e8eddfba6d4d3e07ea5901be2fb025248b and successfully deployed to https://janitor.technology!

**Comment by unicomp21** [comment]
Thanks!
